nilearn/masking.py

```diff
@@ -126,6 +125,8 @@ def intersect_masks(mask_imgs, threshold=0.5, connected=True):
         grp_mask: 3D nibabel.Nifti1Image
             intersection of all masks.
     """
+    from .image import new_img_like
```

Comment:
if I make this import global (that is, only in masking.py), nosetests gives me 15 errors due to circular imports.